frontend/components/News/ArticleRequest.tsx

```diff
@@ -0,0 +1,208 @@
+                    {' '}
+                  </span>
+                  <Link href={routes.member(article.handledBy.id)}>
+                    {getSignature(article.handledBy)}
+                  </Link>
+                  <Typography>
+                    {article.rejectionReason ? (
+                      <>
+                        {t('news:rejectionReason')}
+                        :
+                        {' '}
+                        <Typography fontStyle="italic" component="span">
+                          {article.rejectionReason}
+                        </Typography>
+                      </>
+                    ) : t('news:noRejectionReason')}
+                  </Typography>
+                </Box>
+              </Stack>
+            ) : (hasAccess(apiContext, 'news:article:update')
+            || authorIsUser(article.author, user)) && (
+            <Link style={{ marginLeft: 'auto' }} href={routes.editArticle(article.id)}>
+              <IconButton color="primary">
+                <EditOutlinedIcon />
+              </IconButton>
+            </Link>
+            )}
+
+          </Stack>
+
+          {/* Header */}
+          <Link href={routes.article(article.slug || article.id)}>
+            <Typography variant="h5" className={classes.header}>
+              {selectTranslation(i18n, article.header, article.headerEn)}
+            </Typography>
+          </Link>
+          {(article.imageUrl) && (
+          <div style={{
+            position: 'relative', height: '300px', width: '100%', margin: '1rem 0',
+          }}
+          >
+            <Image
+              layout="fill"
+              src={article.imageUrl}
+              objectFit="cover"
+              style={{
+                borderRadius: '20px',
+              }}
+              alt=""
+            />
+          </div>
+          )}
+          {/* Tags */}
+          {article.tags.length > 0 && (
+            <Box flexDirection="row" flexWrap="wrap">
+              {article.tags.map((tag) => (<Tag key={tag.id} tag={tag} />
+              ))}
+            </Box>
+          )}
+          {/* Body */}
+          <Box
+            ref={markdownRef}
+          >
+            <Markdown content={markdown} />
+          </Box>
+        </Grid>
+        {!rejected && hasAccess(apiContext, 'news:article:manage') && (
+          <Stack
+            direction="row"
+            width="100%"
+            alignItems="center"
+            justifyContent="space-between"
+          >
+            <Button
+              variant="contained"
+              onClick={async () => {
+                await approve();
+                refetch();
+              }}
+            >
+              {t('news:approve')}
+            </Button>
+            <Button
+              variant="contained"
+              color="error"
+              onClick={async () => {
+                const reason = prompt('Why do you want to reject this article?');
```

Comment:
Should replace with MUI Form dialog (https://mui.com/material-ui/react-dialog/#form-dialogs)